### PR TITLE
chore(standard-version): add .versionrc to hide test's section

### PR DESCRIPTION
**Pull request by hiroppy**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?

no

### Motivation / Use-Case

we always delete test's section from CHANGELOG.md manually, but it bothers and this change can prevent to rebase to fix CHANGELOG.md.

### CODE REVIEW

**Comment by evilebottnawi** [comment]
hm, previously standard-release do not generate this stuff, they change logic?

**Comment by hiroppy** [comment]
master

```
$ npx standard-version --dry-run

✔ bumping version in package.json from 3.7.1 to 3.7.2
✔ bumping version in package-lock.json from 3.7.1 to 3.7.2
✔ outputting changes to CHANGELOG.md

---
### [3.7.2](https://github.com/webpack/webpack-dev-server/compare/v3.7.1...v3.7.2) (2019-06-14)


### Bug Fixes

* **open:** set `wait: false` to run server.close successfully ([#2001](https://github.com/webpack/webpack-dev-server/issues/2001)) ([2b4cb52](https://github.com/we
bpack/webpack-dev-server/commit/2b4cb52))
* **test:** fixed ProvidePlugin.test.js ([#2002](https://github.com/webpack/webpack-dev-server/issues/2002)) ([47453cb](https://github.com/webpack/webpack-dev-serve
r/commit/47453cb))


### Tests

* don't use --runInBand and improve execution performance ([#2007](https://github.com/webpack/webpack-dev-server/issues/2007)) ([4dec6b2](https://github.com/webpack
/webpack-dev-server/commit/4dec6b2)), closes [#2005](https://github.com/webpack/webpack-dev-server/issues/2005)
* **e2e:** make browser close before tests run ([#2018](https://github.com/webpack/webpack-dev-server/issues/2018)) ([f611304](https://github.com/webpack/webpack-de
v-server/commit/f611304))
* **options:** set jest.setTimeout ([#1999](https://github.com/webpack/webpack-dev-server/issues/1999)) ([ce81c69](https://github.com/webpack/webpack-dev-server/com
mit/ce81c69))
* **provide-plugin:** use port variable ([#2008](https://github.com/webpack/webpack-dev-server/issues/2008)) ([39b83c4](https://github.com/webpack/webpack-dev-serve
r/commit/39b83c4))
---

✔ committing package-lock.json and package.json and CHANGELOG.md
✔ tagging release v3.7.2
ℹ Run `git push --follow-tags origin master && npm publish` to publish
```

this branch

```
✔ bumping version in package.json from 3.7.1 to 3.7.2
✔ bumping version in package-lock.json from 3.7.1 to 3.7.2
✔ outputting changes to CHANGELOG.md

---
### [3.7.2](https://github.com/webpack/webpack-dev-server/compare/v3.7.1...v3.7.2) (2019-06-14)


### Bug Fixes

* **open:** set `wait: false` to run server.close successfully ([#2001](https://github.com/webpack/webpack-dev-server/issues/2001)) ([2b4cb52](https://github.com/we
bpack/webpack-dev-server/commit/2b4cb52))
* **test:** fixed ProvidePlugin.test.js ([#2002](https://github.com/webpack/webpack-dev-server/issues/2002)) ([47453cb](https://github.com/webpack/webpack-dev-serve
r/commit/47453cb))
---

✔ committing package-lock.json and package.json and CHANGELOG.md
✔ tagging release v3.7.2
ℹ Run `git push --follow-tags origin feature/add-versionrc && npm publish` to publish
```

Test's section is unnecessary.

**Comment by evilebottnawi** [comment]
Interesting, maybe we should add flag, we need investigate why this behavior was changed

**Comment by hiroppy** [comment]
This is a different behavior of standard-version. So we don't need to investigate about this.
https://github.com/conventional-changelog/standard-version/blob/master/.versionrc

File: .versionrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "types": [
+    {"type":"feat","section":"Features"},
+    {"type":"fix","section":"Bug Fixes"},
+    {"type":"test", "hidden": true}
```
Comment (evilebottnawi) [inline .versionrc]: I think we can have extra settings for `Features` and `Bug Fixes` and keep only one line for tests

Comment (hiroppy) [reply .versionrc]: No, they are required.

Comment (evilebottnawi) [reply .versionrc]: hm, okay, i think we need use this:
```
{
  "types": [
    {"type":"feat","section":"Features"},
    {"type":"fix","section":"Bug Fixes"},
    {"type":"test","section":"Tests", "hidden": true},
    {"type":"build","section":"Build System", "hidden": true},
    {"type":"ci","hidden":true}
  ]
}
```

Comment (hiroppy) [reply .versionrc]: Why? We don't use build and CI. They are unnecessary.

Comment (evilebottnawi) [reply .versionrc]: i am use `ci` and `build`, but very rare, it is interesting, why standard-release doesn't generate `chore`, but generate `test:`

Comment (evilebottnawi) [reply .versionrc]: Anyway thanks for investigate, it is really very strange

Comment (hiroppy) [reply .versionrc]: np, I made a mistake to forget to retag when I deleted test's section from CHANGELOG.md and rebased.(v3.6.0)

Comment (hiroppy) [reply .versionrc]: Anyway, we use `ci` and `build` in our repo, please add these. I'll merge this pr.

**Pull request merged**